pkgs/swiftgen/swift2objc/lib/src/parser/parsers/relations_parser.dart

```diff
@@ -0,0 +1,22 @@
+import '../../ast/declarations/compounds/class_declaration.dart';
+import '../_core/type_defs.dart';
+
+class RelationsParser {
+  final JsonMap symbolgraphJson;
+  final DeclarationsMap declarations;
+
+  RelationsParser(this.symbolgraphJson, this.declarations);
+
+  void wireUp() {
+    final List relations = symbolgraphJson["relationships"];
+    for (final relation in relations) {
+      String methodId = relation["source"];
```

Comment:
Check the kind?

Reply:
Added a check to skip relations if `kind != memberOf`. Again, I was just trying to get things to work so I assumed the json is just representing a classes & methods.

Reply:
Getting things working as quick as possible is good. The one important thing to add to the absolute minimum implementation is to make sure it's debuggable. If we have users testing out the package for us, or as we're developing it, we want to be able to diagnose bugs or missing features as quick as possible. So just take the minimal solution and add debuggability and you'll be all good.

Reply:
Got it. I added a log in case we encounter a `kind != memberOf`;